webtorrent/tracker_client.go

```diff
@@ -90,7 +90,7 @@ func (tc *TrackerClient) doWebsocket() error {
 				}
 			case <-closeChan:
 				return
-			default:
```

Comment:
@DigitalAlchemist could you comment on this? I had staticcheck mention this too: There's a loop with a default clause here that could cause busy waiting if neither case is ready. Does it need to be fixed?

Reply:
Merging...